src/compiler/types.extern.js

```diff
@@ -25,5 +25,24 @@
  * The return value is set to *, so that we may optionally return any DOM nodes
  * created during a client-side render. These nodes are often needed for ivars.
  *
- * @typedef {function(!Element):*} BuildDom
+ * @typedef {function(!Element):*}
  */
+
+let BuildDomDef;
+
+/**
+ * See @ampproject/bento-compiler for the definition.
+ * @typedef {*}
```

Comment:
We can include the bento-compiler externs in this project

Reply:
it is written in TS.... we don't have externs

Reply:
Oh...right 😆 We could probably generate externs from that project somehow...not this PR though

Reply:
In a followup PR I'll migrate to `tsc` 🤞

Reply:
Also should these really be externs? I don't get the sense they should. Our convention so far is to have `FooBarDef` defined in code and imported or referenced via path in type annotations. Externs are for global values, properties expected to live on global objects like `window`/`Error` etc., and module stubs